.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,6 @@ Please explain the changes you made here and link to any relevant issues.
 
 ### Checklist
 
-- [ ] `terraform fmt` and `terraform validate` both work from the root and `examples/*` directories
+- [ ] Change added to CHANGELOG.md. All changes must be added and breaking changes and highlighted
 - [ ] CI tests are passing
-- [ ] I've added my change to CHANGELOG.md and highlighted any breaking changes
 - [ ] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation
```

Comment:
doc generation is also tested by the CI https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/.github/workflows/lint.yml#L22-L30. Maybe we should delete this line also ?

Reply:
It's tested, yes, but I think a user still needs to be reminded to how install doc-generation stuff as it's more complicated than fmt/validate.